### ISSUE TITLE
[BUG] Plugin not hooking in WorldGuard

### ISSUE THREAD

**Issue by notedo** [snowgears/shopbugs#393]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of bug

Compatibility issue, Other unexpected behaviour

### Environment

```markdown
- Server Version: git-PaperSpigot-445 (MC: 1.8.8)
- Shop Version: Shop version 1.8.4.2
- WorldGuard Version: WorldGuard version 6.1.3-SNAPSHOT (latest one for 1.8.8)

The plugin lets you create shops even if allow-shop is set to deny in worldguard
https://youtu.be/9GcJx41dfcs

Config: https://mclo.gs/Y5nNwfB
Error in console on startup: https://mclo.gs/0iUH99C
```

### Server startup log

https://mclo.gs/0iUH99C

### Shop config files

https://mclo.gs/Y5nNwfB

### Error log (if applicable)

https://mclo.gs/0iUH99C

### Bug description

The plugin lets you create shops even if allow-shop is set to deny in worldguard
https://youtu.be/9GcJx41dfcs

### Steps to reproduce

set allow-shop to deny
create shop

### Expected behaviour

they shouldnt be able to create shops

### Actual behaviour

they're able to create shops

**Comment by OstlerDev** [comment]
This bug has been verified and fixed and will be released in v1.9.0.2

**Comment by OstlerDev** [comment]
Released 1.9.1.0
